src/vcModals.cpp

```diff
@@ -671,7 +673,11 @@ void vcModals_DrawExportProject(vcState *pProgramState)
 
             ImGui::SetCursorPosX(textAlignPosX);
             if (ImGui::Button(vcString::Get("menuProjectExportButton")))
-              vdkProject_SaveToServer(pProgramState->pVDKContext, pProgramState->activeProject.pProject, udUUID_GetAsString(selectedGroup));
+            {
+              vdkError result = vdkProject_SaveToServer(pProgramState->pVDKContext, pProgramState->activeProject.pProject, udUUID_GetAsString(selectedGroup));
+              if (result != vE_Success)
+                udStrcpy(ErrorText, udTempStr("%s: %s", vcString::Get("errorServerCommunication"), vcProject_ErrorToString(result)));
```

Comment:
`udSprintf(ErrorText, "%s: %s", ...);` would be superior, `udTempStr` is just a wrapper around `udSprintf` with all sorts of additional limitations.